R/tokenizers.R

```diff
@@ -36,7 +37,7 @@ NULL
 #' @rdname tokenize_internal
 #' @importFrom stringi stri_replace_all_regex stri_detect_fixed stri_split_boundaries
 #' @export
-tokenize_word <- function(x, split_hyphens = FALSE, verbose = quanteda_options("verbose")) {
+tokenize_word <- function(x, split_hyphens = FALSE, verbose = quanteda_options("verbose"), ...) {
```

Comment:
What about this: We rename the v3 "word" to "word3" and just make the default through the new options the current version. For compatibility, we could change the tokenizer to a major version number, and not change it during the major version. This could include the sentence rules too.

Reply:
Yes, that the plan. Please do if you want to do now.

Reply:
OK, I'll work on a way to include the "future" v4 tokenizer in a way that can be specified in the options (I love this approach btw!) but still leave the default at v3 so that we can merge and publish it as a v3 release.